src/index.css

```diff
@@ -1,13 +1,68 @@
+:root {
```

Comment:
honestly, not sure how this file got into my PR, but ignore it for now. It doens't do anything too fancy